lib/src/simple_op_store.rs

```diff
@@ -93,10 +105,13 @@ impl SimpleOpStore {
     }
 
     /// Creates an empty OpStore, panics if it already exists
```

Comment:
Minor nit: this comment can be updated since it no longer panics

Reply:
Good catch. Fixed.